topology/intel/dmic-nhlt.h

```diff
@@ -0,0 +1,32 @@
+/*
+  Copyright(c) 2021 Intel Corporation
+  All rights reserved.
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of version 2 of the GNU General Public License as
+  published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful, but
+  WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St - Fifth Floor, Boston, MA 02110-1301 USA.
+  The full GNU General Public License is included in this distribution
+  in the file called LICENSE.GPL.
+*/
+
+#ifndef __DMIC_NHLT_H
+#define __DMIC_NHLT_H
+
+#include <stdint.h>
+
+int set_dmic_vendor_blob(struct tplg_pre_processor *tplg_pp, snd_config_t *cfg,
+			 snd_config_t *parent);
+
+int get_dmic_vendor_blob_size();
+void copy_dmic_vendor_blob(uint8_t *vendor_blob);
```

Comment:
I was actually removing that, because strictly this is not nhlt spec, it is vendor blob. I wanted exactly to avoid confusion of the concepts. OTH I don't mind, so whatever is most clear thing to do